docs/source/tracking.rst

```diff
@@ -202,16 +202,16 @@ An example configuration for a server is as follows:
 
     mlflow server \
         --file-store /mnt/persistent-disk \
-        --artifact-root s3://my-mlflow-bucket/ \
+        --default-artifact-root s3://my-mlflow-bucket/ \
         --host 0.0.0.0
 
 Storage
 ^^^^^^^
 There are two properties related to how data is stored:
 
 * ``--file-store`` a persistent (non-ephemeral) disk where the server stores run and experiment information.
-* ``--artifact-root`` a location suitable for large data (such as an S3 bucket or shared NFS file system)
-  where clients log their artifact output (for example, models). If
+* ``--default-artifact-root`` a location suitable for large data (such as an S3 bucket or shared NFS file system)
+  where clients log their artifact output (for example, models) for newly created experiments. If
   you do not provide this option, clients write artifacts to *their* local directories,
```

Comment:
This line I think is a bit confusing ("clients write artifacts to *their* local directories"), in that the client will actually write to the artifact directory given by the experiment corresponding to the current run (so a client could log artifacts to multiple different directories if a program launches multiple runs). It could be nice to explicitly state at the top of this section that the root artifact URI is a property of the experiment, e.g maybe we add a subsection that conveys some of the following info (copied from a comment on #158):

> ### Artifact Storage
>
> The root artifact URI for a run is determined at the experiment level. That is, runs created under an experiment will derive their artifact URI from that of the experiment, and the root artifact URI for an experiment can be specified at experiment-creation time e.g. via `mlflow experiments create --artifact-root s3://<my-bucket>`. Currently, a run will use the artifact URI \<artifact-root\>/\<experiment-id\>/\<run-id\>.
>
> Launching mlflow server --default-artifact-root s3://\<some-bucket\> will launch the tracking server configured to use s3://\<some-bucket\> as the root artifact directory for any new experiments created against that tracking server. However, it won't modify the metadata (e.g. artifact URI) of existing experiments.


(also as a sanity check, the logic for determining a run's artifact URI is [here](https://github.com/smurching/mlflow/blob/4246cbe6ffd972d5f6824db63a9dea5f1b0f27f5/mlflow/store/file_store.py#L73))